torchft/process_group.py

```diff
@@ -670,6 +698,15 @@ def __init__(self, timeout: timedelta = timedelta(seconds=60.0)) -> None:
 
         self._errored: Optional[Exception] = None
 
+        NONBLOCKING_TIMEOUT_ENV = "TORCH_NCCL_NONBLOCKING_TIMEOUT"
```

Comment:
just confirming, this env variable is read by pytorch before the collective is launched and not when `ProcessGroupNCCL` is initialized right?

Reply:
Yes, it's read on the first collective